src/mtcp/mtcp_restart.c

```diff
@@ -672,7 +740,6 @@ main(int argc, char *argv[], char **environ)
     total_reserved_fds = reserve_fds_upper_half(reserved_fds);
 
     // Refer to "blocked memory" in MANA Plugin Documentation for the addresses
-#ifdef GNI
 # define GB (uint64_t)(1024 * 1024 * 1024)
     // FIXME:  Rewrite this logic more carefully.
     char *start1, *start2, *end1, *end2;
```

Comment:
I'm curious when will we go this routine?
`(libsEnd + 1 * GB < highMemStart)`

Reply:
this happens on Cori.